e2e_tests/tests/per_provider/normal_tests/key_attestation.rs

```diff
@@ -135,12 +136,29 @@ mod activate_credential {
         client
             .generate_rsa_sign_key(key_name.clone())
             .expect("Failed to generate key");
+        let prep_activ_cred = client
+            .prepare_activate_credential(key_name.clone())
+            .expect("Failed to get parameters for MakeCredential");
+
+        let (cred, secret) = make_credential(prep_activ_cred);
+
+        // Wrong `secret` value
+        let error = client
+            .activate_credential(key_name.clone(), vec![0xDE; 52], vec![0xAD; 256])
+            .unwrap_err();
+        assert_eq!(error, ResponseStatus::PsaErrorInvalidArgument);
+
+        // Wrong `credential` size (after decryption)
+        let error = client
+            .activate_credential(key_name.clone(), vec![0xDE; 52], secret)
+            .unwrap_err();
+        assert_eq!(error, ResponseStatus::PsaErrorInvalidArgument);
 
-        let _error = client
-            .activate_credential(key_name, vec![0xDE; 52], vec![0xAD; 256])
+        // Wrong `secret` value
```

Comment:
Yeah, they actually complain about the same thing - I just wanted to test when `credential` both is and isn't valid, just that it eventually doesn't matter because ActivateCredential fails on `secret` anyway.